Source/RapyutaSimulationPlugins/Public/Drives/RobotVehicleMovementComponent.h

```diff
@@ -16,82 +20,128 @@
 
 #include "RobotVehicleMovementComponent.generated.h"
 
+/**
+ * @brief Type of odometry frame origin.
+ * World provide odometry from world origin and Encoder provide odometry from initial pose.
+ */
 UENUM(BlueprintType)
 enum class EOdomSource : uint8
 {
     WORLD UMETA(DisplayName = "World"),
     ENCODER UMETA(DisplayName = "Encoder")
 };
 
+/**
+ * @brief Base Robot vehicle movement class which is used as part of #ARobotVehicle. 
+ * Robot moves based on the Velocity(member of UMovementComponent) and #AngularVelocity without considering physics.
+ * If Robot bumped into something, try to slide along it.
+ * 
+ * If #bFollowPlatform is true, robot will follow the pawn movement under the robot with tag "ContactPoints", which is useful for elevator and etc.

```

Comment:
```suggestion
 * If #bFollowPlatform is true, robot will follow the pawn movement under the robot which has been defined as the #MovingPlatform (e.g. elevators), it will also adapt its pose to the floor surface configuration (e.g. slopes)

```